### ISSUE TITLE
ansible-test sanity throws error on helm 'import outside toplevel'

### ISSUE THREAD

**Issue by geerlingguy** [ansible-collections/community.kubernetes#132]
##### SUMMARY

When running `ansible-test sanity` locally in my Python 3.7 environment, I get the following error:

```
Running sanity test 'pylint' with Python 3.7
ERROR: Found 1 pylint issue(s) which need to be resolved:
ERROR: plugins/modules/helm.py:345:12: import-outside-toplevel: Import outside toplevel (tempfile)
```

##### ISSUE TYPE

- Bug Report

##### COMPONENT NAME

helm

##### ANSIBLE VERSION

```paste below
ansible 2.9.9
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/Users/jgeerling/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.7 (default, Apr  9 2020, 17:13:06) [Clang 11.0.3 (clang-1103.0.32.29)]
```

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

macOS

##### STEPS TO REPRODUCE

Run the following in this repository:

```yaml
ansible-test sanity
```

##### EXPECTED RESULTS

Tests pass.

##### ACTUAL RESULTS

See summary.

**Comment by geerlingguy** [comment]
Looks like the issue is in:

```python
    if release_values != {}:
        try:
            import tempfile
        except ImportError:
            module.fail_json(
                msg=missing_required_lib("tempfile"),
                exception=traceback.format_exc(),
                stdout='',
                stderr='',
                command='',
            )
```

**Comment by geerlingguy** [comment]
This was fixed in #115.
